### DISCUSSION TITLE
PM can't generate and appy configs after VM migration

### DISCUSSION THREAD

**Discussion by ankost403** [peering-manager/peering-manager#894] [Q&A] [answered]
Hi, our peering-manager was migrated to a new VM and after that it freezes with message "Working..." when I go to Configuration. Job status is Pending, start and stop time is empty. The only thing I can do with these jobs is to delete them. However if I click "Raw" when watching "Working..." then configuration is displayed and it looks correct. I suspected a connectivity issue, found and fixed ssh key permissions, but it didn't help. Please advise how to troubleshoot this problem.

**Comment by gmazoyer** [comment]
Make sure HTTP requests performed in the background can properly reach API endpoints. This should be observable in your browser debug console. This is most likely due to an HTTP/HTTPS misconfiguration if you changed your frontend server / reverse proxy.

**Reply by ankost403** [reply]
Thanks for the reply. In the network activity console I see that after opening router configuration page, it sends every second a request /api/core/jobs/698/ and receives code 200. In the response I see: 
status: "pending", 
started: null, 
completed: null
This is as I understand just a polling for the job status, but how can I get what's happening under the hood?

**Reply by gmazoyer** [reply, marked as answer]
Make sure that the rqworker process [has been setup](https://docs.peering-manager.net/setup/3-peering-manager/) and is running so it can process jobs.

**Reply by ankost403** [reply]
Found that peering-manager-rqworker service was off. After enabling and starting it, PM's operation normalzied. Thank you for your help.
